iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonParser.m

```diff
@@ -0,0 +1,159 @@
+//
+//  MSGeoJsonParser.h
+//  MSMapsModules
+//
+//  Created by Elizabeth Bartusiak (t-elbart) on 6/12/20.
+//  Copyright © 2020 Microsoft Corporation.
+//  Licensed under the MIT license.
+//
+
+#import "MSMapGeoJsonParser.h"
+#import <MicrosoftMaps/MicrosoftMaps.h>
+
+@implementation MSMapGeoJsonParser {
+  MSMapElementLayer *layer;
+};
+
++ (MSMapElementLayer *)parse:(NSString *)geojson error:(NSError **)error {
+  if (geojson == (id)[NSNull null] || geojson.length == 0) {
+    *error = [NSError
```

Comment:
Error should be nullable so check against nil before assigning to it.

Reply:
One way to simplify the error == nil option that the public API should have is to have a local NSError that you pass to internalParse and then copy back to the output error before returning.